helm/hpcc/values.schema.json

```diff
@@ -1111,6 +1111,16 @@
         "disabled": {
           "type": "boolean",
           "description": "If true, disable OTel based trace/span generation"
+        },
+        "logSpanStart": {
```

Comment:
I'm not inclined to add more options than necessary and would encourage us to avoid these maybe condense into a single "logSpan"

With that said, if we keep both of these, I wonder how useful logging only one is w/out adding timing info into the payload (I still contend a single log output at end w/ all pertinent info is most useful)

Reply:
I think we should default logSpanStart to off now (and I will amend the PR).  I agree that a single log entry at then end is most useful, and I was going to disable tracing at the start when I create a PR to improve the finish logging.  Given the amount of tracing in esp I think it would be better to disable now.